### DISCUSSION TITLE
59

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#722] [Comments]
http://jojoldu.tistory.com/59

**Comment by jojoldu** [comment]
작성자: Kingbbode

항상 감탄하고갑니다! 후기 잘 봤습니다. 감사합니다!

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

안녕하세요 kingbbode님^^
제가 오히려 더 많이 보고 배우고 있습니다.
항상 방문해주셔서 감사합니다!

**Comment by jojoldu** [comment]
작성자: fastutle

좋은 후기 감사드립니다.

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

좋게 봐주셔서 제가 더 감사드립니다^^ 자주 방문 부탁드리겠습니다!

**Comment by jojoldu** [comment]
작성자: 김용주

항상 okky를 통해서 여러 후기들 잘 보고 있습니다. 감사합니다.

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

앗 ㅎㅎ;
제가 더 감사드립니다!
더 열심히 후기 작성하겠습니다^^

**Comment by jojoldu** [comment]
작성자: 강청

^^ 좋은후기 고맙습니다~ 
간접적으로나마 경험이 되네요

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

좋은 답변에 제가더 감사드립니다^^
자주 방문 부탁드립니다!

**Comment by jojoldu** [comment]
작성자: Chan

늦었지만, 후기 잘 보았습니다. ^_^

**Comment by gonghojin** [comment]
오늘도 잘 읽고 갑니다. 창천항로님이랑 같은 세미나를 들은 적이 있는데(4월 비전공 si탈출 관련), 정리하신 세미나 후기 내용이 세미나 강의 내용과 하나도 빠짐없이 다 정리되있더라고요.
그후 창천항로님 세미나 후기는 현장 참여 느낌(?)으로 읽게 되네요.. 감사합니다~

**Comment by hanyeonhee** [comment]
뭐 검색하다가 옛날에 쓰신 글 발견했는데...얼른 집에가서 오늘의 커밋을 해야겠다! 라니 ㅋㅋ 넘 큐티...
